docs/End-User-Guide.md

```diff
@@ -309,6 +315,20 @@ Click the **Device** icon and select the desired device from the drop-down menu.
 
 <img src="images/survey-creator-preview-different-devices.png" alt="Survey Creator - Preview the survey on different devices" width="75%">
 
+## Themes Tab
+
+The Themes tab enables you to customize the applied UI theme. You can change colors, sizes, fonts, shadow effects, and other survey appearance parameters in a user-friendly interface.
+
+<img src="images/theme-editor.png" alt="Survey Creator - The Themes tab" width="100%">
+
+### How to: Save a Custom Theme
+
+SurveyJS themes are JSON objects with CSS variables and other theme settings. Once you finish customization, click the Export button to download a JSON object with your custom theme.
+
+<img src="images/theme-export.png" alt="SurveyJS Theme Export">
+
+Other capabilities depend on features implemented by your development team. For instance, the following help topic describes how developers can save themes on a server and load them from it: [Save and Load Custom Themes](/survey-creator/documentation/theme-editor#save-and-load-custom-themes).
+
```

Comment:
### How to: Load an Existing Theme

You can also load a theme from a file. Click the Import button to open a file which contains a custom theme.

<img src="..." alt="SurveyJS Theme Import>